conda/notices/types.py

```diff
@@ -76,8 +77,15 @@ def _parse_iso_timestamp(iso_timestamp: Optional[str]) -> Optional[datetime]:
 
     @classmethod
     def get_cache_key(cls, url: str, name: str, cache_dir: Path) -> Path:
-        """Returns the place where this channel response will be stored as cache"""
+        """
+        Returns the place where this channel response will be stored as cache
```

Comment:
Not following the "Return the place" instead of "Returns the place" grammar?

Reply:
(A pep 257 style guide https://peps.python.org/pep-0257/#one-line-docstrings)

Reply:
The sentence is written with as if it begins with "This function..." So, fully spelled out:

> This function returns the place where this channel response will be stored as cache

I've always been pretty liberal with my usage of sentence fragments in doc strings, but would be open to changing that habit if it makes it more clear.

Reply:
Not a big deal